concepticondata/conceptlists/Williamson-1973-118.tsv

```diff
@@ -0,0 +1,119 @@
+ID	NUMBER	ENGLISH	FRENCH	GERMAN	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Williamson-1973-118-1	1	animal	animal	Tier	619	ANIMAL
+Williamson-1973-118-2	2	arm: hand	bras: main	Arm: Hand	2121	ARM OR HAND
+Williamson-1973-118-3	3	armpit	aisselle	Achselhöhle	1886	ARMPIT
+Williamson-1973-118-4	4	arrow	flèche	Pfeil	977	ARROW
+Williamson-1973-118-5	5	ashes	cendres	Asche	646	ASH
+Williamson-1973-118-6	6	ask (a question)	demander	fragen	1309	ASK (INQUIRE)
+Williamson-1973-118-7	7	bad, be	mauvais (être)	schlecht sein	1292	BAD
+Williamson-1973-118-8	8	bean	haricot	Bohne	832	BEAN
+Williamson-1973-118-9	9	beg (=ask for something)	demander (qqc à qqn)	bitten um, erbitten, betteln	1828	ASK (REQUEST)
+Williamson-1973-118-10	10	belly	ventre	Bauch, Unterleib	1251	BELLY
+Williamson-1973-118-11	11	bite	mordre	beissen	1403	BITE
+Williamson-1973-118-12	12	buffalo	buffle	Büffel	934	BUFFALO
+Williamson-1973-118-13	13	build	bâtir	bauen	1840	BUILD
+Williamson-1973-118-14	14	calabash	calebasse	Kalebasse	411	GOURD
+Williamson-1973-118-15	15	carve	sculpter	schnitzen	228	CARVE
+Williamson-1973-118-16	16	charcoal (coal)	charbon	Holzkohle (Kohle)	2659	CHARCOAL OR COAL
+Williamson-1973-118-17	17	child	enfant	Kind	2099	CHILD
+Williamson-1973-118-18	18	come out	sortir	herauskommen	1107	GO OUT
+Williamson-1973-118-19	19	cool, be(come)	se refraîchir	kühl sein, kühl werden		
```

Comment:
3068 COOL DOWN?

Reply:
3068 COOL DOWN is transitive, i.e. to make something cool down, while here it's intransitive.

Reply:
I wouldn't necessarily pinpoint transitivity to the *definition* of a gloss. *Cool down* can be used in either case (The weather cooled off, She cooled down the tea by mixing it with cold water).

Reply:
Then ditransitivity of the concept should be reflected in the definition to allow the possibility of covering a wider ranger of concepts, right? Otherwise, we wouldn't have DRY UP and MAKE DRY, for example, if we'd say that DRY theoretically can be used in both ways.

Reply:
But is this a case for transitivity/intransitivity? *dry up* can be used either way (Dry up the dishes, the lake dried up) and we find both, transitive and intransitive usages, mapped to it in Concepticon.

Reply:
So in that case I'd think, since COOL DOWN doesn't necessarily specify transitivity (in my opinion, but I might simply be wrong here), it can be mapped to either case.

Reply:
Hm if that's the case then I might have treated this pair of concepts wrong from the very beginning. But it would also be difficult for me to distinguish between the two then because I simply wouldn't understand in which relationship they are to each other conceptually and hierarchically, especially if a gloss in a hypothetical list is ambiguous.